ci/prometheus-down.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# 
+# This script checks a few known queries against the internal prometheus query
+# API.  These queries are chosen because they have timestamps as their values.
+# 
+# If the query fails, then something is wrong with Prometheus's query API.
+# 
+# If at least one timestamp is up to date, then we know it's probably working,
+# otherwise, something is jammed up inside probably.
+# 
+# 
+# XXX If something stops the processes that generate these timestamp metrics,
+# this will result in false positives.  Maybe a deploy might do this?
+#
+
+set -ux
+
+QUERIES="
+  concourse_has_opslogin_lastcheck
+  bosh_unknown_iaas_instance_lastcheck
+  bosh_last_scrape_timestamp
+  aws_iam_user_lastcheck
+"
+TIME=`date +%s`
+APIOK=yes
+UPDATEOK=no
+
+for QUERY in ${QUERIES} ; do
+  QTIME=`curl --max-time 5 -s 0.prometheus.production-monitoring.prometheus-production.toolingbosh:9090/api/v1/query?query=${QUERY} \
+   | jq -r '.data.result[0].value[1]' | sed 's/\..*//'`
+
+  # make sure that the curl worked (indicates that prometheus is down entirely)
+  if [ -z ${QTIME} ] ; then
+    APIOK=no
+  else
+    # make sure that the data is not too old (indicates that prometheus is not accepting data)
+    TIMEDIFF=`expr ${TIME} - ${QTIME}`
```

Comment:
Is it possible for `QTIME` to be something other than empty or an integer?

Reply:
I don't think so.  I tried a few bad things (strings and floats), and it seemed to either give out an int, or nothing.